tests/unit/client/test_dataframe_client.py

```diff
@@ -9,14 +9,18 @@ def test_properties(project_client):
     parent = project_client
 
     domain_dataframe = domain.Dataframe(
-        description="some description", tags=["x"], name="test title"
+        description="some description",
+        tags=["x"],
+        comments=["this is a comment"],
+        name="test title",
     )
```

Comment:
there's a `test_properties` in each of the client entities' unit tests - can we update them all like this?

looks like you've got dataframe and experiment - so we'll need artifact, feature, metric and parameter too